supplier_calendar/models/res_partner.py

```diff
@@ -31,8 +31,6 @@ def supplier_plan_days(self, date_from, delta):
         :return: datetime: resulting date.
         """
         self.ensure_one()
-        if isinstance(delta, float):
-            delta = round(delta)
```

Comment:
ping @LoisRForgeFlow

Reply:
Thanks @HviorForgeFlow 

Those lines are on purpose and need to be there. (ref: https://github.com/OCA/purchase-workflow/pull/2255)

Please @bizzappdev do not remove it

Reply:
I added it back. @LoisRForgeFlow